src/js/helpers/cleanData.js

```diff
@@ -288,8 +291,7 @@ export const cleanLinks = (links, pageType) => {
     const link = edge.node;
```

Comment:
are we still using all the pageType specific logic above this?

Reply:
the clean links pagetype logic?

Reply:
yeah, we have stuff like 

```
  // Topics
  if (pageType === 'topic') {
    for (const edge of links.edges) {
      const link = edge.node;
      if (link.topiccollections && link.topiccollections.edges.length) {
        for (const edge of link.topiccollections.edges) {
          const { topiccollection } = edge.node;

          link.topLinks = [];
          link.otherLinks = [];
          link.url = `${pathPrefix || ''}/${link.slug}`;
          link.text = link.title;
          link.topiccollection = topiccollection;

          cleanedLinks.push(link);
        }
      }
    }

    return cleanedLinks;
  }
```

it seems like a good place to look for future cleanup

Reply:
ive cleaned up some, some of them are still being used. there are probably some i missed, or other refactoring we could do.

Reply:
made an issue for future cleanup https://github.com/cityofaustin/techstack/issues/4330